policy/predicate/file.go

```diff
@@ -140,6 +192,14 @@ func (exp ComparisonExpr) MarshalText() ([]byte, error) {
 	return []byte(fmt.Sprintf("%s %d", op, exp.Value)), nil
 }
 
+func (exp ComparisonExpr) MarshalTextToString() string {
```

Comment:
This should be called `String` to match Go conventions (implement the `Stringer` interface)